AgentDiscoveries-Backend/src/main/resources/db/migration/V1__add_initial_tables.sql

```diff
@@ -8,13 +7,14 @@ create table user (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 create table agent (
-  `agent_id` int NOT NULL AUTO_INCREMENT,
+    `call_sign` varchar(20) NOT NULL,
+    `user_id` int NOT NULL,
   `first_name` varchar(30),
   `last_name` varchar(100),
-  `date_of_birth` date NOT NULL,
+  `date_of_birth` date,

```

Comment:
Why is this stuff nullable? Is that allowed?

Reply:
I made D.O.B nullable because it seemed inconsistent to complain about that when the name columns were null. But you're right that doing the inverse would have made more sense..